src/main/java/org/mifos/processor/bulk/connectors/service/ProcessorStartRouteService.java

```diff
@@ -248,6 +250,7 @@ public void startBatchProcessCsv(Exchange exchange) throws IOException {
         variables.put(PAYER_IDENTIFIER_VALUE, exchange.getProperty(PAYER_IDENTIFIER_VALUE));
         variables.put(REGISTERING_INSTITUTE_ID, exchange.getProperty(REGISTERING_INSTITUTE_ID));
         variables.put(IS_FILE_VALID, true);
+        variables.put(PAYEE_DFSP_ID, payeeDFSPId);
```

Comment:
use this naming convention only for API header. don't use it for Zeebe variable

Reply:
Done